src/main/java/org/commonjava/indy/service/tracking/model/dto/TrackedContentEntryCutDTO.java

```diff
@@ -0,0 +1,180 @@
+/**
+ * Copyright (C) 2023 Red Hat, Inc. (https://github.com/Commonjava/indy-tracking-service)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.commonjava.indy.service.tracking.model.dto;
+
+import org.commonjava.indy.service.tracking.model.AccessChannel;
+import org.commonjava.indy.service.tracking.model.StoreEffect;
+import org.commonjava.indy.service.tracking.model.StoreKey;
+
+import java.util.Set;
+
+public class TrackedContentEntryCutDTO
```

Comment:
Maybe we could call this as "TransferDTO" as it is just used in content service? We can give a javadoc to show why this DTO exists, like:

This DTO is a cut version of the TrackedContentEntryDTO, because it is just used to reduce the payload size for interacting with content service.

Reply:
updated accordingly